tests/conftest.py

```diff
@@ -58,6 +58,10 @@ def cluster(request):
         msg = "The given cluster can not be connected to: {}. ".format(cluster_path)
         msg += "Provide a valid --cluster-path or use --deploy to deploy a new cluster"
         pytest.fail(msg)
+    elif not system.is_path_empty(cluster_path) and deploy:
+        msg = "The given cluster path is not empty: {}. ".format(cluster_path)
```

Comment:
you can just format the string like

```python
msg = (
    f"The given cluster path is not empty: {cluster_path}. "
    f"Provide an empty --cluster-path and --deploy to deploy a new cluster"
)
```
This is the preferred way from our contribution [docs](https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md).

Don't you want to mention why the directory has to be empty? Is it cause you would like to prevent to override the data which was generated via the installer and prevent of their lost?

Reply:
@petr-balogh sure thing, I'll format it that way next time. Would you like me to modify this message here? I'm thinking perhaps the docs can explain more about this?

Reply:
I don't want to block this PR, if it's ready for merge and verified please merge this. And docs can be improved in some next PR if you feel it's worth to mention it.